FirebaseTwitterAuthUI/FirebaseTwitterAuthUI.h

```diff
@@ -0,0 +1,27 @@
+//
+//  Copyright (c) 2016 Google Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+#import <UIKit/UIKit.h>
+
+//! Project version number for FirebaseTwitterAuthUI.
+FOUNDATION_EXPORT double FirebaseTwitterAuthUIVersionNumber;
+
+//! Project version string for FirebaseTwitterAuthUI.
+FOUNDATION_EXPORT const unsigned char FirebaseTwitterAuthUIVersionString[];
+
+#import <FirebaseTwitterAuthUI/FIRTwitterAuthUI.h>
```

Comment:
`FIRTwitterAuthUI.h` needs to be marked as a Public header (not Project header) or it'll cause build errors when integrating as a framework.